commit 41c3e9873cf2b388d4a91fbd86c360d53b1a00da
Author: Matthias Clasen <mclasen@redhat.com>
Date:   Sun Mar 21 20:56:17 2021 -0400

    Move compose related tooling to a subdir
    
    This reduces the clutter in gtk/.

diff --git a/gtk/compose-parse.py b/gtk/compose/compose-parse.py
similarity index 100%
rename from gtk/compose-parse.py
rename to gtk/compose/compose-parse.py
diff --git a/gtk/gtk-compose-lookaside.txt b/gtk/compose/gtk-compose-lookaside.txt
similarity index 100%
rename from gtk/gtk-compose-lookaside.txt
rename to gtk/compose/gtk-compose-lookaside.txt
